ui-participant/src/hub/HubPage.test.tsx

```diff
@@ -57,6 +57,14 @@ jest.mock('../providers/UserProvider', () => {
   }
 })
 
+jest.mock('providers/I18nProvider', () => {
```

Comment:
let's make a test-util for this so we don't have to repeat it everywhere.  Something like `useMockI18n(mockTexts)` .  And this pattern might be so common we might just want to make a "renderWithI18n" helper that takes the component and an (optional) list of message keys, similar to how we have renderWithRouter helpers.